### DISCUSSION TITLE
Question about rootRulesFolder

### DISCUSSION THREAD

**Discussion by ryuseongryong** [jertel/elastalert2#1244] [Q&A] [answered]
Hello

I'm using elastalert2 in kubernetes for elasticsearch alerting free.
(Install helm on k8s as `helm install` cmd)

I have question about latest update rootRulesFolder.
rootRulesFolder is explained as the "`folder must be located in the root of the chart directory.`"
Does it mean using elatstalert2 by recreate chart personally?
Or is there any other ideas?

Thank you

**Comment by jertel** [comment]
UPDATE: The following statements are incorrect per the comment below: "You should be able to specify an absolute directory for that value and the chart will attempt to find rule files within it. The "must be located in the root" I believe refers to the default _relative_ directory 'rules', which has a couple of sample rules that ship with the chart."

@mehrdad-khojastefar, since you added this feature it would be good to hear your thoughts on what you expected individuals to do when they use the pre-bundled chart that is available on the official chart repository: https://jertel.github.io/elastalert2

**Reply by ryuseongryong** [reply]
thanks.

**Comment by mehrdad-khojastefar** [comment, marked as answer]
Since we are using `.Files` object in the rules configMap we only have access to files in our chart folder. 
There are some workarounds but personally I don't recommend them since this way is cleaner. [More info on this topic](https://github.com/helm/helm/issues/4026#issuecomment-387904230)

Personally I would recommend to write your rules inside rules folder in the pre-bundled chart, but you should be okay to place them anywhere else as long as it is in your chart folder.

**Reply by ryuseongryong** [reply]
thanks.
